src/Model/Serializer/ValueObject/Taxonomy/Category/CategoryDenormalizer.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Model\Serializer\ValueObject\Taxonomy\Category;
+
+use CultuurNet\UDB3\Model\ValueObject\Taxonomy\Category\Category;
+use CultuurNet\UDB3\Model\ValueObject\Taxonomy\Category\CategoryDomain;
+use CultuurNet\UDB3\Model\ValueObject\Taxonomy\Category\CategoryID;
+use CultuurNet\UDB3\Model\ValueObject\Taxonomy\Category\CategoryLabel;
+use Symfony\Component\Serializer\Exception\UnsupportedException;
+use Symfony\Component\Serializer\Normalizer\DenormalizerInterface;
+
+final class CategoryDenormalizer implements DenormalizerInterface
+{
+    private ?CategoryDomain $defaultDomain;
+
+    public function __construct(?CategoryDomain $defaultDomain = null)
+    {
+        $this->defaultDomain = $defaultDomain;
```

Comment:
There are several legacy classes that need to be removed, namely `EventType`, `Facility` and `Theme`. They all derive from `Category` and have a domain of `eventtype`, `facility` or `theme`. In the new model layer, we only have one `Category` and one denormalizer. But now this denormalizer also needs to be used for older data that was lacking the domain.

Once we are on a common interface this can be improved.